src/main/java/com/sparkplatform/api/models/DateField.java

```diff
@@ -22,6 +22,9 @@ public DateField(String str) {
 			}
 		}
 	}
+	public DateField() {
+		value = null;
+	}
```

Comment:
All of these `*Field.java` models were getting snagged [here](https://github.com/sparkapi/SparkJava/blob/master/src/test/java/com/sparkplatform/api/core/PropertyAsserter.java#L216), as they don't have a default constructor.  Since the fields can be `null` from the API, it seemed fitting to allow the default constructor without a value.

Admittedly, these are the changes I'm least certain about.